examples/gnrc_border_router/README.md

```diff
@@ -33,21 +195,21 @@ the program by calling something like:
 ```bash
 cd dist/tools/tunslip
 make
-sudo ./tunslip6 affe::1/64 -t tun0 -s /dev/ttyUSB0
+sudo ./tunslip6 2001:db8::1/64 -t tun0 -s /dev/ttyUSB0
 ```
 Assuming that `/dev/ttyUSB0` is the device descriptor for the (additional) UART
 interface of your RIOT board.
 
 On the RIOT side you have to configure the SLIP interface by configuring a
 corresponding IPv6 address, e.g.
 ```
-ifconfig 6 add affe::2
+ifconfig 6 add 2001:db8::2
 ```
 and adding the SLIP interface to the neighbor cache (because Linux won't
 respond to neighbor solicitations on an interface without a link-layer address)
 by calling
 ```
-ncache add 6 affe::1
+ncache add 6 2001:db8::1
 ```
 After this you're basically done and should be able to ping between the border
 router and the outside world (assuming that the Linux host is properly
```

Comment:
To me there is a step missing here (and some more detail as well), because at this point the prefix is not propagated to the motes behind the BR (that was the case for me):
* detail that interface `6` is the wired interface and `5` is the wireless one, just for clarity
* configure the wireless interface with a global address using the same prefix:
```
ifconfig 5 add 2001:db8::3
```